docs/misc/logging.conf

```diff
@@ -0,0 +1,23 @@
+[loggers]
+keys=root
+
+[handlers]
+keys=hand01
+
+[formatters]
+keys=form01
+
+[logger_root]
+level=NOTSET
+handlers=hand01
+
+[handler_hand01]
+class=FileHandler
+level=INFO
+formatter=form01
+args=('borg.log', 'w')
```

Comment:
so it will just put borg.log into current directory?

can you practically try this (just to avoid that it might reject relative pathes)?